From cf6db125f271b1dd9d79f9a9549c75c038fd6134 Mon Sep 17 00:00:00 2001
From: Andrew Jeffery <andrew@aj.id.au>
Date: Tue, 5 May 2020 12:15:12 +0930
Subject: [PATCH 140/147] README: Add description of install targets

The Makefile has been updated with ``install`` and ``install-user``
targets. Mention them in the README to save those tinkering with the
``develop`` branch from manually running the instructions.

Signed-off-by: Andrew Jeffery <andrew@aj.id.au>
---
 README.rst | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/README.rst b/README.rst
index 6d53dae..e4368d0 100644
--- a/README.rst
+++ b/README.rst
@@ -68,7 +68,17 @@ Build a fresh distribution package::
 
 This will create a distributable archive located in the ``dist/`` folder.
 
-Installing directly from file::
+
+*Shortcut on ``develop``*
+
+If you are using the ``develop`` branch since May 2020, you can run ``make
+install-user`` to install your current working branch in your user environment
+or ``make install`` for a system-wide installation. The ``DESTDIR`` variable
+can be provided to ``make install`` to adjust the base installation path (it
+defaults to ``DESTDIR=/usr/local``).
+
+Otherwise, on other branches, follow the steps below to manually install the
+distribution archive::
 
     # Build
     make dist
-- 
2.31.1

